res/controllers/Denon-MC7000-scripts.js

```diff
@@ -562,6 +563,11 @@ MC7000.PadButtons = function(channel, control, value, status, group) {
                 if (engine.getValue("[Sampler" + i + "]", "track_loaded") === 0) {
                     engine.setValue("[Sampler" + i + "]", "LoadSelectedTrack", 1);
                 } else if (engine.getValue("[Sampler" + i + "]", "track_loaded") === 1) {
+                    // stop playing all other samplers ...
+                    for (j = 1; j <=8; j++) {
+                        engine.setValue("[Sampler" + j + "]", "cue_gotoandstop", 1);
+                    }
+                    // ... before the actual sampler to play gets started
```

Comment:
I do not like this feature addition. Its uncommon and restrictive....

Reply:
Can you explain why you think this feature is a good addition?

Reply:
Indeed, this behavior might be surprising. Sampler decks should act independently.

Reply:
I love it this way. I use jingles for radio stream. I just cannot play around with the samples as I like when I use the default ment method to stop a sampler with shift + pad button. Either I need to let a sampler finish until to trigger a new one or they play on top of each other which sounds bad most time. 

2nd experience: some weeks ago I had a private gig for a singer/songwriter friend and his label guys. So he gave me some vocals and short samplers to use while I play my set. Same story. I could play half the sampler, trigger a new one to create a "new" combined sampler. Playing on top of each other would have been a desaster. I couldn't have stopped the playing one with shift + pad while starting the new one with another pad button same time on time. 

I gonna keep it this way. it is by far not restrictive but much more interesting and creative things possible.

You don't need to like my mapping. I am happy to keep it for myself and close the PR.

Before one suggests again to keep my own modified version and submit another one to Mixxx... No, this is starting with only few little things but going to end in 2 completely different versions of the same mapping and I don't have time and energy for doubling my work on this.

Don't get me wrong guys. I so much appreciate your input and work you do for this project. Also most of your suggestions to modify my submissions  makes absolutely sense. So thank you again for all your reviews and time you spend. Cheers! 

Have a great day!

Reply:
Interesting. Thanks for sharing that. I think that seems reasonable. We may consider implementing as a feature in C++.

Reply:
In case I could add a new user variable, default setting to stop a sampler but one could easily change to previous behavior when (s)he wants to play samplers on top of each other. This would allow both possibilities in one mapping. Just let me know

Reply:
Sure, a customizable constants in the top section of the script would work. The default behavior in the official mapping should not cause side effects between different sampler decks.

Reply:
Customizable mappings with a preferences UI would be nice, but that's unrealistic right now.

Reply:
> Interesting. Thanks for sharing that. I think that seems reasonable. We may consider implementing as a feature in C++.

I agree! This sounds like a valid use case that is worth considering, independent of any mapping. If it's part of the application we could also add options to configure the personal preferences.

Reply:
I agree, its a valid usecase and should be implemented in mixxx itself so its available for all mappings instead of just being a controller mapping hack.

Reply:
In the meantime, it would be ok to make the behavior configurable using another variable at the top of the script.

Reply:
I added a user variable accordingly - documentation will be updated during tomorrow.